Rollbar/RollbarEventArgs.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace Rollbar
+{
+    using Newtonsoft.Json;
+    using Rollbar.DTOs;
+    using System;
+    using System.Text;
+
+    /// <summary>
+    /// An abstract base for implementing concrete Rollbar EventArgs.
+    /// </summary>
+    /// <seealso cref="System.EventArgs" />
+    /// <seealso cref="Rollbar.ITraceable" />
+    public abstract class RollbarEventArgs 
+        : EventArgs
+        , ITraceable
+    {
+        private RollbarEventArgs()
+        {
+
+        }
+
+        protected RollbarEventArgs(
+            RollbarConfig config, 
+            Payload payload
+            )
+        {
+            this.Config = config;
+            if (payload != null)
+            {
+                this.Payload = JsonConvert.SerializeObject(payload);
+            }
+            //else
```

Comment:
Do we want to ensure that this.Payload is always not null? Or should we just remove this comment?